src/main.rs

```diff
@@ -49,7 +49,7 @@ use alacritty::util::fmt::Red;
 fn main() {
     // Load command line options and config
     let options = cli::Options::load();
-    let config = load_config(&options);
+    let config = load_config(&options).flag(&options);
```

Comment:
I think you're missing the override for live config update.

Reply:
@chrisduerr fixed, though I'm starting to wish `Config` was smarter. Somehow knowing what values have been flagged, and avoiding the call to flag them again in the live reload section.

But for now my last commit seems fine, and works as intended.

Reply:
I think there's quite a bit of yak-shaving to be done with the configuration system of alacritty. Eventually I'll get around and try to make at least a little PoC crate that handles all relevant configuration file wishes. :)

Reply:
I like that idea. Yea reading through the config stuff is a bit crazy right now. Partly because of serde.

Reply:
Serde is doing an amazing job at deserialization, it's just not made to never fail deserialization. Something wrapping around serde is probably a better choice.

Reply:
I wasn't saying serde is bad, yea it's just a good idea to wrap the stuff dealing with it.